cli/reporter/output_handler_test.go

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) Mondoo, Inc.
+// SPDX-License-Identifier: BUSL-1.1
+
+package reporter
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestOutputHandlerAwsSqs(t *testing.T) {
+	validSqsUrls := []string{
+		"https://sqs.us-east-1.amazonaws.com/123456789012/MyQueue",
+		"http://sqs.us-east-1.amazonaws.com/123456789012/MyQueue",
+		"https://sqs.eu-central-1.amazonaws.com/123456789012/MyQueue",
+		"http://sqs.eu-central-1.amazonaws.com/123456789012/MyQueue",
+		"sqs.us-east-1.amazonaws.com/123456789012/MyQueue",
+		"sqs.eu-central-1.amazonaws.com/123456789012/MyQueue",
+	}
+
+	for i, sqsUrl := range validSqsUrls {
+		rep, err := NewOutputHandler(sqsUrl, "JSON")
+		require.NoError(t, err, i)
+		require.IsType(t, &awsSqsHandler{}, rep, i)
+	}
+
+	invalidSqsUrls := []string{
+		"https://sqss.us-east-1.amazonaws.com/123456789012/MyQueue",
+		"http://sqss.us-east-1.amazonaws.com/123456789012/MyQueue",
+		"sqss.us-east-1.amazonaws.com/123456789012/MyQueue",
+		"sqss.eu-central-1.amazonaws.com/123456789012/MyQueue",
+		"sqs.europe-central-1.amazonaws.com/123456789012/MyQueue",
+		"somethingtotallyrandom",
+	}
+
+	for i, sqsUrl := range invalidSqsUrls {
+		_, err := NewOutputHandler(sqsUrl, "JSON")
+		require.Error(t, err, i)
+	}
+}
+
+func TestOutputHandlerLocal(t *testing.T) {
+	validFileTarget := []string{
+		"file:///root/test",
```

Comment:
Do you think ` --output json --output-target report.json` should not work?

Reply:
I even used `--output json --output-target file://report.json` and it has not stored the report in the current directory.

Reply:
I just tested it, work fine for me
```
~/go/bin/cnspec scan --output json --output-target file://report.json
....
→ wrote report to file file=report.json
```

Are you seeing any errors?

Reply:
No error, it just prints the json output on cli